src/lib/tsconfig-tests.json

```diff
@@ -11,7 +11,15 @@
     "experimentalDecorators": true,
     "strictNullChecks": true,
     "paths": {
-      "@angular/cdk": ["../../dist/packages/cdk"],
+      "@angular/cdk/a11y": ["../../dist/packages/cdk/a11y"],
+      "@angular/cdk/bidi": ["../../dist/packages/cdk/bidi"],
+      "@angular/cdk/coercion": ["../../dist/packages/cdk/coercion"],
+      "@angular/cdk/platform": ["../../dist/packages/cdk/platform"],
+      "@angular/cdk/portal": ["../../dist/packages/cdk/portal"],
+      "@angular/cdk/keyboard": ["../../dist/packages/cdk/keyboard"],
+      "@angular/cdk/rxjs": ["../../dist/packages/cdk/rxjs"],
+      "@angular/cdk/table": ["../../dist/packages/cdk/table"],
+      "@angular/cdk/observe-content": ["../../dist/packages/cdk/observe-content"],
```

Comment:
I think here without the `public_api` this can be just `@angular/cdk/*: ["../../dist/packages/cdk/*"]`